jose-b64/src/zero.rs

```diff
@@ -5,6 +5,7 @@
 
 use core::ops::{Deref, DerefMut};
 
+#[allow(dead_code)]
 pub trait Zeroize {
     fn zeroize(&mut self);
 }
```

Comment:
Seems like this can just be deleted if unused. It duplicates `zeroize::Zeroize` anyway.

Reply:
Yes, it's a duplicate, but it's used in some cases when the `secret` feature is not enabled. However, it is also intertwined with the `serde` feature.
To my understanding, the full matrix is:
 * `secret` and `serde` -> `zeroize` crate
* `!secret` and `serde` -> this placeholder is defined and used
* `secret` and `!serde` -> this placeholder is not defined
* `!secret` and `!serde` -> this placeholder is defined but not used

The warning only happens in the last case.